src/core.c/CompUnit/Repository/Installation.pm6

```diff
@@ -88,7 +89,13 @@ sub MAIN(:$name, :$auth, :$ver, *@, *%) {
             $!installed-dist;
         }
 
-        method meta(--> Hash:D) {
+        method !repo-dist {
+            ⚛$!repo-dist // cas $!repo-dist, {
+                $_ // CompUnit::Repository::Distribution.new(self)
```

Comment:
Does this really need an instance?

Reply:
No, but for caching purposes. A trade-off.

Reply:
If you mean the attribute.

Reply:
After b92ba9af8ff903da5c6da4a04ab3d40c49d219ef where `module-dependency` was removed, the attribute became basically redundant and has been removed in d6b4b5f160dc3e060d2d9beb510a986362bdc88e.